src/pages/workspaces/List.js

```diff
@@ -165,30 +161,18 @@ const WorkspaceCard = pure(({
     span({ style: { color: colors.dark(0.85) } }, ['No description added'])
   const titleOverrides = !canView ? { color: colors.dark(0.7) } : {}
 
-  const renderCard = children => {
-    const style = listView ? styles.longCard : styles.shortCard
-    if (canView) {
-      return a({
-        href: Nav.getLink('workspace-dashboard', { namespace, name }),
-        style
-      }, children)
-    } else {
-      return h(Clickable, {
-        as: 'a',
-        onClick: onRequestAccess,
-        style
-      }, children)
-    }
-  }
-
   return h(TooltipTrigger, {
     content: !canView && `
       You cannot access this workspace because it is protected by an Authorization Domain.
       Click to learn about gaining access.
     `,
     side: 'top'
   }, [
-    renderCard([
+    h(Clickable, {
+      href: canView ? Nav.getLink('workspace-dashboard', { namespace, name }) : undefined,
+      onClick: !canView ? onRequestAccess : undefined,
```

Comment:
This bugs me a little. If the user doesn't have access, this will render as a `div` whereas all others are rendered as `a`. It seems to render fine in Chrome, but this is inconsistent both in the tag and block/inline rendering.

That said, it feels to me like these should all be `div`s. What would you think of adding `as: 'div'`?

Reply:
Only `a` tags allow navigation by `href`. And strictly speaking, `a` without `href` is not valid (though it's such a common mistake that all browsers accept it). Mixed tags seem ok here, and they'll render the same since they're all flex children.